Mm/blogpost06222020

MissyM2 opened the pull request:
The blog looks complete.  I tested the images and the links.  They work.  I am pinging @RedNumberIX , though.  He seems to be linking to a presentation but there is only an individual image available.

BarakStout commented:
@MissyM2 two things: 
- Can we update the name of this PR to something more insightful?
- Can we remove the `Gemfile.lock` from the PR? We can open a separate PR to update it if needed

MissyM2 commented:
@BarakStout 

- I updated the title.  If you have a standard you want me to follow on the blogposts, let me know.  I just gave it the name of the branch.  
- copied over the master branch Gemfile.lock into my branch.
- updated the blog with the correct presentation from @RedNumberIX

MissyM2 commented:
OK, the file name referenced in the file and the actual filename match in capitalization.  I pushed it up.  It looks fine on local and see the screenshot but when I push, it just says 'everything up to date' and I do not see the change in the md file on the server. 
<img width="839" alt="Screen Shot 2020-06-22 at 11 58 07 AM" src="https://user-images.githubusercontent.com/34319929/85308797-af1d4a00-b47f-11ea-8e7b-4faef0020a80.png">
